### ISSUE TITLE
Central-ledger /documentation & /swagger.json end-points are failing

### ISSUE THREAD

**Issue by mdebarros** [mojaloop/project#997]
**Summary**:
Central-ledger `/documentation` & `/swagger.json` resource end-points are failing

**Severity**:
Low

**Priority**:
Low

**Expected Behavior**
1. `/documention` resource should render the Swagger Documentation page/portal correctly allowing users to explore and interact with the API. 
2. `/swagger.json` resource should provide a copy of the generate swagger file that defines the central-ledger admin API.

**Steps to Reproduce**
  1. Start central-ledger locally and access either one of the following end-points:
  2. http://localhost:3001/documentation
  3. http://localhost:3001/swagger.json

Alternatively, if you can access it on dev2:
- http://xxxx-central-ledger.mojaloop.live/documentation
- http://xxxx-central-ledger.mojaloop.live/swagger.json

**Specifications**
  - Component (if known): central-ledger
  - Version: v8.1.1
  - Platform: all
  - Subsystem: swagger
  - Type of testing: manual
  - Bug found/raised by: @mdebarros 

**PRs**
- [ ] https://github.com/mojaloop/central-ledger/pull/478

**Notes**:

- Severity when opened:
- Priority when opened:

**Comment by ggrg** [comment]
It seems like bad definition(s) in `src/api/participants/routes.js` are breaking hapi-swagger

**Comment by lewisdaly** [comment]
Switching to the branch in the above PR fixes the issues with swagger not loading, but since it's not yet fully compatible with Joi 16, there may be some other issues downstream.

**Comment by lewisdaly** [comment]
Look like the latest release of `hapi-swagger` `v10.3.0` still hasn't fixed the issue. But it looks like the above pull request is close to being completed.

**Comment by lewisdaly** [comment]
Release `v11.0.0` includes the above PR that introduces hapi 16.0.0, so that should do the trick.

**Comment by lewisdaly** [comment]
@mdebarros @rmothilal can we verify that this is fixed now? Or are we still waiting on a release?

**Comment by amarmodus** [comment]
@lewisdaly - Tested in **Dev2**

central-ledger.mojaloop.live/documentation is ok

central-ledger.mojaloop.live/**swagger**.json throws following error
{
    "errorInformation": {
        "errorCode": "2001",
        "errorDescription": "Internal server error - Cannot read property 'properties' of undefined"
    }
}

**Comment by amarmodus** [comment]
central-ledger.mojaloop.live/documentation and 
central-ledger.mojaloop.live/swagger.json - both ok in Dev 2 

but the ml-api-adaptor/swagger is giving  following error
{
    "errorInformation": {
        "errorCode": "2001",
        "errorDescription": "Internal server error - Cannot mix different versions of joi schemas"
    }
}

**Comment by lewisdaly** [comment]
That's a shame. Thanks for checking the `ml-api-adapter` as well. I guess that's a different issue, so maybe we'll close that one and open a new ticket for ml-api-adapter. Hopefully the fix will just be in fixing up the package.

**Comment by lewisdaly** [comment]
Follow up ticket for the `ml-api-adapter` is here: https://github.com/mojaloop/project/issues/1120
